src/wanna/core/templates/sla_cloud_function.py

```diff
@@ -0,0 +1,31 @@
+import json
+from datetime import datetime as dt
+
+from google.cloud import aiplatform_v1, storage
+
+
+def main(event, context):
+    path = event["id"].rsplit("/", 1)[0].split("/", 1)[1]
+    storage_client = storage.Client()
+    bucket = storage_client.get_bucket(event["bucket"])
+    blob = bucket.blob(path)
+    data = blob.download_as_text().splitlines()
+
+    client = aiplatform_v1.PipelineServiceClient(
+        client_options={"api_endpoint": "europe-west1-aiplatform.googleapis.com"},
+    )
+
+    for line in data:
+        entry = json.loads(line)
+        timestamp = entry["jsonPayload"]["startTime"]
+        project_id = entry["logName"].split("/")[1]
+        location = entry["resource"]["labels"]["location"]
+        pipeline_id = entry["resource"]["labels"]["pipeline_job_id"]
+        name = f"projects/{project_id}/locations/{location}/pipelineJobs/{pipeline_id}"
+        delta = dt.now() - dt.strptime(timestamp[:-8].replace("T", " "), "%Y-%m-%d %H:%M:%S")
+
+        request = aiplatform_v1.GetPipelineJobRequest(name=name)
+        pipeline = client.get_pipeline_job(request=request)
+        if delta.total_seconds() > 3600 * int(pipeline.labels["wanna_sla"]):  # 60 seconds * 60 minutes
```

Comment:
In our yaml grammar we call this sla_hours, right ?

Lets keep it clear this value is in hours, rename the label to `wanna_sla_hours. 

As an extra can I use 1.5 hours in our yaml ? it seems I can't since you cast it to int. Likely in the yaml grammar you also use int, lets make it a float. WDYT